src/main/java/uk/gov/justice/hmpps/offenderevents/subscribe/PrisonerEventsListener.java

```diff
@@ -24,6 +25,8 @@ public class PrisonerEventsListener {
     private final Duration totalDelay;
     private final Duration delay;
 
+    private static final List<String> DELAYED_EVENT_TYPES = List.of("OFFENDER_MOVEMENT-RECEPTION", "OFFENDER_MOVEMENT-DISCHARGE", "BOOKING_NUMBER-CHANGED");
+
```

Comment:
I don't think BOOKING_NUMBER-CHANGED needs to be delayed, only the ones that result in the receive and release events

Reply:
ok ill remove this next time.